### PR TITLE
ci: update jenkins config to use shared ci pipeline

### LINKED ISSUE #3: update jenkins config to use shared pipeline

### PR DESCRIPTION
**Pull request by edaniszewski**
fixes #3

### CODE REVIEW

File: .jenkins [hunk 1]
```diff
@@ -1,42 +1,19 @@
...
-                sh 'docker run --rm -v ${WORKSPACE}:/repo edaniszewski/ghr -u ${GITHUB_USER} -r ${GITHUB_REPONAME} -t ${GITHUB_TOKEN} -replace ${TAG_NAME} dist/'
-            }
-        }
-
-    }
-}
\ No newline at end of file
+pythonPipeline([
+  'pythonVersion': '3.8',
+  'twineCredential': 'pypi-token-fake-rpigpioo-upload',
```
Comment (lazypower) [inline .jenkins]: just making sure this isn't a typo.

**Review by lazypower — APPROVED** [review]

File: .jenkins (hunk 1 above)
Comment (lazypower) [reply .jenkins]: looks like it is.

`pypi-token-fake-rpigpio-upload`

Comment (edaniszewski) [reply .jenkins]: nice catch  -- thanks!

Comment (edaniszewski) [reply .jenkins]: This should now be fixed

**Pull request merged**